docker-compose.yml

```
@@ -0,0 +1,85 @@
+      db:
+        condition: service_healthy
+    networks:
+      - app-network
+
+  
+  nginx:
+    image: nginx:stable-alpine
+    container_name: business_care_nginx
+    restart: unless-stopped
+    ports:
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)